eq-author-api/migrations/addVersion.js

```diff
@@ -0,0 +1,6 @@
+//This is an auto-generated file.  Do NOT modify the method signature.
+
+module.exports = questionnaire => {
+  questionnaire.version = 1;
```

Comment:
Do we want to be defensive about this? Or are we happy making the assumption that the version number is the source of truth? e.g. should this be `questionnaire.version = questionnaire.version || 1` or should we error/log a warning if it already has a version?

This is more of an open question that a request to change the above BTW.

Reply:
I think we have to assume each migration moves from one state to another. The state before (now) has no version so this adds it. I would be worried where the one with a version got it from.

Reply:
When is it not undefined? Surely this migration would not have been run in that instance?

Reply:
As discussed, we will leave this untouched and always set to version 1.